CHANGES.rst

```diff
@@ -46,6 +46,11 @@ badpix_selfcal
 - Subtract pedestal dark when constructing min array across selfcal exposures
   for MIRI MRS data. [#8771]
 
+calwebb_coron3
+--------------
+
+- Tighten tolerance of psf alignment. [#8717]
```

Comment:
Technically, you're loosening the tolerance...

Reply:
The tolerances for the fit are tighter. ftol and xtol for [leastsq](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.leastsq.html#leastsq) default to 1.49012e-08. This PR tightens them to 1E-15.

The tolerances for the tests are loosened but as that's a test change I didn't note it in the changelog.